docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -226,22 +233,25 @@ The options are:
 * *log* - The Connectors instance sends errors to its log.
 * *dead letter queue* - The Connectors instance sends messages that it cannot handle to a dead letter topic that you define for the Connectors Kafka instance.
 +
-For example, select *log*.
+For example, accept the default *stop* option.
 
 . Click *Next*.
 
 . Review the summary of the configuration properties and then click *Create Connectors instance*.
 +
-Your Connectors instance is listed in the table of Connectors. After a couple of seconds, the status of your Connectors instance changes to the *Ready* state and it starts producing messages and sending them to its associated Kafka topic.
+Your Connectors instance is listed in the table of connectors. After a couple of seconds, the status of your Connectors instance changes to the *Ready* state and it starts producing messages and sending them to its associated Kafka topic.
 +
 From the connectors table, you can stop, start, and delete your Connectors instance, as well as edit its configuration, by clicking the options icon (three vertical dots).
 
 .Verification
-ifdef::qs[]
-* Did you create an instance of the Data Generator connector?
-endif::[]
 
-In the next procedure, you can verify that the source Connectors instance is sending messages as expected by creating a sink Connectors instance that consumes the messages.
+* Does your source Connectors instance generate messages?
+
+.. In the OpenShift Application Services web console's left navigation bar, select *Applications and Data Services* > *Streams for Apache Kafka* > *Kafka Instances*.
```

Comment:
For consistency with how we've written this type of step elsewhere, I suggest one of the following:

```
In the Hybrid Cloud Console, select *Applications and Data Services* > *Streams for Apache Kafka* > *Kafka Instances*.
```

```
In the OpenShift Application Services web console, select *Streams for Apache Kafka* > *Kafka Instances*.
```

```
In the {product-kafka} web console, click *Kafka Instances*.
```